### PR TITLE
l10n: removal of unused property

### PR DESCRIPTION
**Pull request by miurahr**
Remove unused properties from Bundle.properties

## Pull request type

- Other (describe below)
refactor

## What does this PR change?

- Removed;
    - JAXB loading error message in Java 1.5 age
    - TMX error message in OmegaT 1.4 age
    - Removed GUI labels
- Unify Unicode escape rule(//u01AB) as same as English for all L10N files  
    - When merging #737 L10N project can produce  standard L10N  escape

### CODE REVIEW

**Review by Kazephil — APPROVED** [review]
Looks good to me.

I like the addition of comments indicating which file uses the preferences.

File: src/org/omegat/Bundle.properties [hunk 1]
```diff
@@ -191,9 +191,6 @@ BUTTON_FILTER=&Filter
 BUTTON_REPLACE=&Show Replacements
 BUTTON_REPLACE_ALL=Re&place All
 
-BUTTON_SELECT=&OK
-BUTTON_SELECT_NO_MNEMONIC=Select
```
Comment (miurahr) [inline src/org/omegat/Bundle.properties:195]: these two is observed in ProjectPropertiesDialog.java and SearchWIndowContoller.java as commented out form.

**Comment by miurahr** [comment]
> Looks good to me.
> 
> I like the addition of comments indicating which file uses the preferences.

These items are unused, so no file is related with the preferences, except for two above items that are commented out in the file.

File: src/org/omegat/Bundle.properties [hunk 2]
```diff
@@ -822,9 +819,6 @@ PP_SENTENCE_SEGMENTING=Se&ntence-level segmenting
 
 PP_EDIT_PROJECT=Project Properties
 
-# TODO REMOVE THIS
-PP_MESSAGE_EDITPROJ=
```
Comment (miurahr) [inline src/org/omegat/Bundle.properties:826]: This part was changed in  015c041d33d63a4feea397197cb34a41a615b059 by @Kazephil

File: src/org/omegat/Bundle.properties [hunk 3]
```diff
@@ -1158,10 +1152,6 @@ LFC_ERROR_FILE_DOESNT_EXIST=File {0} does not exist!
 
 PM_ERROR_SAVE=Preferences system save operation failed!
 
-PM_WARNING_PARSEERROR_ON_READ=Parse error encountered reading preferences file!
-
-PM_UNSUPPORTED_ENCODING=Unsupported file encoding
```
Comment (miurahr) [inline src/org/omegat/Bundle.properties:1163]: Last modified in 377d7485682d1f2f75b48dc70e742090a712a387  in 2006 which was used in Preferences.java long time ago.

File: src/org/omegat/Bundle.properties [hunk 4]
```diff
@@ -1206,10 +1196,6 @@ TMXR_WARNING_WHILE_PARSING=Warning on line {0}, column {1} while parsing:
 # the error from library follows
 TMXR_FATAL_ERROR_WHILE_PARSING=Fatal error while parsing "project_save.tmx" file(line {0}, column {1}):
 
-TMXR_ERROR_TU_BEFORE_HEADER=Translation unit encountered before TMX header.
-
-TMXR_WARNING_TUV_LANG_NOT_SPECIFIED=Language not specified for translation unit variant. Variant skipped.
```
Comment (miurahr) [inline src/org/omegat/Bundle.properties:1211]: Last modified in 2006 in 767563e36618272c9a24ed7cbd7534fb4df75b49

File: src/org/omegat/Bundle.properties [hunk 5]
```diff
@@ -2169,7 +2152,6 @@ MATCHES_OPEN_PREFERENCES=TM Matching Options
 DICTIONARY_HIDE=Hide '{0}'
 
 STARTUP_ERRORBOX_TITLE=Error on application startup
-STARTUP_JAXB_LINKAGE_ERROR=Either update to the latest JDK 1.6, or remove the jaxb-*.jar files from the OmegaT /lib dir
```
Comment (miurahr) [inline src/org/omegat/Bundle.properties:2172]: Last modified in 2022 by @Kazephil  and discussed the message mensions too old Java version.

**Review by miurahr — COMMENTED** [review]
note some messages origins last modifications

**Review by brandelune — APPROVED** [review]

**Pull request merged**